### PR TITLE
Updated developing docs with pkg creation

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

**Comment by sascala** [comment]
I think I addressed most of the questions we discussed, though 3) and 5) could probably use more explanation.

1) what is the Universe?
2) what are packages?
3) how should i version my packages?
4) how should i distribute my application?
5) how do i specify optional configuration?

File: 1.8/development/create-package.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+* `config.json` - Configuration properties supported by the package, represented as a json-schema.
+* `marathon.json.mustache` - A mustache template that, when rendered, creates a Marathon app definition capable of running your service.
+
+# `package.json`
+
+Every package in Universe must have a `package.json` file that specifies the highest-level metadata about the package (comparable to a package.json in Node.js or setup.py in Python).
+
+Currently, a package can specify one of two values for `.packagingVersion`, either 2.0 or 3.0. The version declared will dictate which other files are required for the complete package as well as the schemas all the files must adhere to.
+
+Consider the following guidelines when creating a your `package.json` file:
```
Comment (BenWhitehead) [inline 1.8/development/create-package.md]: Confusing: `a your`

I imagine it should be either "when creating a `package.json` file" or "when creating your `package.json` file"

File: 1.8/development/create-package.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        "23b1cfe8e04a": "some-org/foo:1.0.0"
+      }
+    }
+  }
+}
+```
+
+# `config.json`
+
+This file declares the packages configuration properties, such as the amount of CPUs, number of instances, and allotted memory.  The defaults specified in `config.json` will be part of the context when `marathon.json.mustache` is rendered.
```
Comment (BenWhitehead) [inline 1.8/development/create-package.md]: I think it'd be good to include a note about config.json being a json schema and provide a link to the json schema docs.

In universe we have this as the leading sentence:
`This file describes the configuration properties supported by the package, represented as a
[json-schema](http://spacetelescope.github.io/understanding-json-schema/)`

File: 1.8/development/index.md [hunk 3]
```diff
@@ -4,85 +4,13 @@ nav_title: Development
 menu_order: 4
 ---
 
-## <a name="universe"></a>Package Repositories
+# <a name="universe"></a> DC/OS Universe
```
Comment (BenWhitehead) [inline 1.8/development/index.md]: `s/DC/OS Universe/Mesosphere Universe/g` There isn't a DC/OS Universe, Mesosphere hosts and runs the Universe, it just so happens DC/OS comes pre-configured to point to the Mesosphere Universe.

Think of this as following the pattern of Android. The Android OS itself does not provide an app store, Google provides the Play Store.

File: 1.8/development/index.md [hunk 4]
```diff
@@ -4,85 +4,13 @@ nav_title: Development
...
-* The `postInstallNotes` parameter gives the user information they'll need after the installation. Focus on providing a documentation URL, a tutorial, or both. For example: `"postInstallNotes": "Thank you for installing the Unicorn service.\n\n\tDocumentation: http://<service-url>\n\tIssues: https://github.com/"`
-* The `postUninstallNotes` parameter gives the user information they'll need after an uninstall. For example, further cleanup before reinstalling again and a link to the details. A common issue is cleaning up ZooKeeper entries. For example: `postUninstallNotes": "The Unicorn DC/OS Service has been uninstalled and will no longer run.\nPlease follow the instructions at http://<service-URL> to clean up any persisted state" }`
-
  [1]: http://mesosphere.github.io/universe/
- [2]: http://mesosphere.github.io/universe/#publish-a-package-1
- [3]: /docs/1.8/administration/securing-your-cluster/
- [4]: https://github.com/mesosphere/universe
- [5]: http://mesosphere.github.io/universe/#submit-your-package
- [6]: https://github.com/mesosphere/dcos-helloworld
+ [2]: http://mesosphere.github.io/universe/#publish-a-package-1
```
Comment (BenWhitehead) [inline 1.8/development/index.md]: Updated URL: `https://github.com/mesosphere/universe#publish-a-package-1`

File: 1.8/development/create-package.md [hunk 5]
```diff
@@ -0,0 +1,153 @@
+---
+post_title: Creating a Universe Package
+nav_title: Creating a Universe Package
+menu_order: 10
+---
+
+This page covers general advice and information about creating a DC/OS Universe package. Consult the [Publish a Package][2] page of the Universe documentation for full details.
```
Comment (BenWhitehead) [inline 1.8/development/create-package.md]: `s/DC/OS Universe package/DC/OS package that can be published to the Mesosphere Universe/`

**Comment by ssk2** [comment]
cc @ryadav88

**Comment by emanic** [comment]
Sorry @sascala just seeing this, there are some conflicts unfortunately. Could you resolve and then will be more quick next time thanks

**Pull request merged**